crawler/db/mysql.py

```diff
@@ -251,8 +239,9 @@ def reset_is_current_flags(cursor: CMySQLCursor, rna_ids: List[str], chunk_size:
 
     total_rows_affected = 0
     for rna_ids_group in rna_ids_groups:
-        rna_ids_sql = format_sql_list_str(rna_ids_group)
-        cursor.execute(SQL_MLWH_MARK_ALL_SAMPLES_NOT_MOST_RECENT % rna_ids_sql)
+        in_array = ",".join(["%s"] * len(rna_ids_group))
+        sql_statement = SQL_MLWH_MARK_ALL_SAMPLES_NOT_MOST_RECENT % ("%s", in_array)
+        cursor.execute(sql_statement, [datetime.now()] + rna_ids_group)
```

Comment:
Turns out when you have two insertion points in the SQL template, you have to do this massive hoop jumping exercise.  I tried using named insertions, but they only made the situation worse.  This is the only thing I could find that works.  It basically replaces the second argument with a template for insertion, like `%s,%s,%s,%s` for four values.  Then when we execute the SQL, we pass in the IDs we want to look for and they get slotted into our ad-hoc template.

Reply:
I had similar problems with this, good you found a better way of doing it